pkg/test/aws/3.19.0/aws_internet_gateway/input.json

```diff
@@ -3,7 +3,6 @@
   "Id": "igw-0184eb41aadc62d1c",
   "Type": "aws_internet_gateway",
   "Attrs": {
-   "arn": "arn:aws:ec2:eu-west-3:047081014315:internet-gateway/igw-0184eb41aadc62d1c",
```

Comment:
You should keep in there in `input.json` files, if not we couldn't assert that it has been removed in the `expected.json`

Input are mostly raw output from the enum lib (tf providers) without any normalization.